baseband/mark4/base.py

```diff
@@ -186,8 +196,8 @@ def find_header(self, template_header=None, ntrack=None, decade=None,
             Number of tracks used to store the data.  Required if
             ``template_header`` is ``None``.
         decade : int, optional
-            Decade the observations were taken (needed to remove ambiguity in
-            the Mark 4 time stamp).  Required if ``template_header`` is
+            Decade the observations were taken in, needed to remove ambiguity
```

Comment:
I think we should either remove the `decade` option, or add the `ref_time` argument.